Makefile

```diff
@@ -198,6 +204,16 @@ generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 		output:webhook:dir=./config/webhook \
 		webhook
 
+.PHONY: generate-kv-go-client
+generate-kv-go-client: $(CLIENT_GEN) ## Generate kubevirt client-go for e2e
+	$(CLIENT_GEN) \
+		--input-base="kubevirt.io/api" \
+		--input="core/v1" \
+		--output-pkg="sigs.k8s.io/cluster-api-provider-kubevirt/e2e/generated/client-go/clientset" \
+		--clientset-name="versioned" \
+		--output-dir /home/nunnatsa/GIT/cluster-api-provider-kubevirt/e2e/generated/client-go/clientset \
+		--fake-clientset=false
+
```

Comment:
do we need the kubevirt client once you transition e2e to use virtctl?

Reply:
I'll remove kv client. this is still WIP.